app/views/sidebar/_tos.html.haml

```diff
@@ -11,35 +11,18 @@
         %ol
           %li
             %h3
-              Acceptance of Terms
+              RELEASE AND WAIVER OF LIABILITY AGREEMENT
             %p
-              I hereby state that I am voluntarily participating in the Adopt a Drain volunteer program and that I am 18 years of age or older. I have carefully read this agreement, and fully understand its contents. I am aware that this contains a release of liability and I am signing it of my own free will.
+              I acknowledge that I intend to voluntarily participate in the Adopt A Drain program on property owned by the City of Savannah.
             %p
-              I have read, understand, and agree to follow the Volunteer Guidelines on safety and debris removal stated above. I hereby grant permission to Adopt a Drain to use photographs of me taken during this activity on its website and in other publications, at Adopt a Drain's sole discretion and without any consideration.
+              I AM AWARE THAT I COULD SUSTAIN SERIOUS INJURY OR EVEN DEATH.  I AM VOLUNTARILY PARTICIPATING IN THIS ACTIVITY WITH KNOWLEDGE OF THE DANGER INVOLVED, AND AGREE TO ASSUME ANY AND ALL RISKS OF BODILY INJURY, DEATH OR PROPERTY DAMAGE, WHETHER THOSE RISKS ARE KNOWN OR UNKNOWN.
             %p
-              I also agree that I will notify Savannah Public Works immediately via the Adopt-a- Drain website if, at any point, I wish to discontinue my service as an Adopt-a-Drain volunteer.
+              As consideration for being permitted by the Mayor and Aldermen of the City of Savannah (the “City”) and the Savannah Water Resources Division to participate in the Adopt A Drain activity on City owned property, I forever release the City and the Water Resources Division, any affiliated organization, their respective directors, officers, employees, volunteers, agents, contractors, and representatives (collectively “Releasees”) from any and all actions, claims, or demands that I, my assignees, heirs, distributees, guardians, next of kin, spouse and/or legal representatives now have, or may have in the future, for injury death, or property damage, related to (i) my participation in this activity; (2) the negligence or other acts, whether directly connected to this activity or not, and however caused, by any Releasee, or (iii) the condition of the premises where this activity occurs, whether or not I am then participating in the activity.  I also agree that I, my assignees, heirs, distributees, guardians, next of kin, spouse and legal representatives may not make a claim against, sue, or attach the property of any Releasee in connection with any of the matters covered by the foregoing Release.
```

Comment:
Change to the bullet points:
"(i) my participation in this activity; **(2)** the negligence or other acts, whether directly connected to this activity or not, and however caused, by any Releasee, or (iii) the condition of the premises where this activity occurs, whether or not I am then participating in the activity."

To:
"(i) my participation in this activity; **(ii)** the negligence or other acts, whether directly connected to this activity or not, and however caused, by any Releasee, or (iii) the condition of the premises where this activity occurs, whether or not I am then participating in the activity."